app/assets/stylesheets/category.scss

```diff
@@ -0,0 +1,122 @@
+$italianno: 'Italianno', cursive;
+$italiana: 'Italiana', serif;
+$roboto: 'Roboto', sans-serif;
+$blue: #3778c2;
```

Comment:
- [ ] I would recommend using the same font which is recommended in the[ design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=).

Reply:
Thank you for the feedback, I have implemented this change 👍